### ISSUE TITLE
bug in TestClass.throws

### ISSUE THREAD

**Issue by KoenT-IM** [Steve-Fenton/tsUnit#10]
It looks like the "throws" statement in TestClass is not working.
The following code works fine (test succeeds because exception is thrown):
```typescript
    goodTest5()
    {
        this.throws(() =>
        {
            console.log("Exception thrown. Test should succeed.");
            throw new Error("exception!")
        }, "Code should throw an exception.");
    }
```
However, the following code does not work as expected (test still succeeds, even though no exception is thrown):
```typescript
    badTest5()
    {
        this.throws(() =>
        {
            console.log("No exception thrown here. Test should fail!");
        }, "Code should throw an exception.");
    }
```
Expected behavior: badTest5 should be a failed test.
Observed behavior: badTest5 ends up in the passed tests.

**Comment by Steve-Fenton** [comment]
All fixed :)

**Comment by Steve-Fenton** [comment]
Released in 2.0.1 and available on NuGet, and NPM.

**Comment by KoenT-IM** [comment]
Thanks Steve!
